db/schema.rb

```diff
@@ -312,6 +312,7 @@
     t.string "slug"
     t.string "status", default: "active"
     t.datetime "updated_at", null: false
+    t.index ["slug"], name: "index_chat_channels_on_slug", unique: true
```

Comment:
accidental addition?

Reply:
I'm not sure: this index keep re-appearing ever time I run migrations. Hypershield doesn't appear to let me see the underlying structure of a table: can you tell me if this index is in production @mstruve?

Reply:
@joshpuetz If you consistently have this problem, drop the index from your local DB and it will stop reappearing in the diffs.

Reply:
@citizen428 This index appears even after I drop my entire database!